src/app/core/rpc/rpc-state.class.ts

```diff
@@ -13,30 +13,12 @@ export class RPCStateClass {
     this.rpc.registerStateCall('getnetworkinfo', 10000);
     this.rpc.registerStateCall('getstakinginfo', 10000);
 
-    this.lastBlockTimeState();
     this.blockLoop();
     this.walletLockedState();
     this.coldStakeHook();
     this.initWalletState();
   }
 
-  private lastBlockTimeState() {
```

Comment:
This code calls getblockchaininfo everytime it needs to... you can put this code back and the blocks will update as required